packs/cloudflare/tests/test_action_get_ips.py

```diff
@@ -0,0 +1,128 @@
+import yaml
+import requests
+import requests_mock
+
+from st2tests.base import BaseActionTestCase
+
+from get_ips import GetIPs
+
+__all__ = [
+    'GetIPsTestCase'
+]
+
+MOCK_CONFIG_BLANK = ""
+
+MOCK_CONFIG_NO_SCHEMA = """
+api_host: "api.cloudflare.com"
+"""
+
+MOCK_CONFIG_FULL = """
+api_host: "mock://api.cloudflare.com"
+"""
+
+MOCK_DATA_INVALID_JSON = "{'dd': doo}"
+
+MOCK_DATA_SUCCESS = """
```

Comment:
Thanks, 

I've followed the suggestion and raised #493 to move the fixtures into separate files.